packages/palette/src/elements/Menu/Menu.tsx

```diff
@@ -14,20 +14,24 @@ import { Sans } from "../Typography"
 interface MenuProps {
   children?: React.ReactNode
   onClick?: (event: React.MouseEvent<HTMLElement>) => void
+  margin?: SpaceProps["m"]
```

Comment:
I'm not exactly sure how these palette components are normally set up, but instead of having custom `margin` and `padding` here, could we have the `Menu` component accept `m` and `p` and pass those in directly (with some defaults)?

Reply:
Do you mean remain the `margin` and `padding` props to `m` and `p` respectfully?

These props currently have default values, but I can rename.

Reply:
I realize I made this comment before my second one 🙈.

It looks like the `margin` value here changes the outermost margin on the component, so renaming to `m` makes sense to me.

It seems like `padding` is a separate case!